ko/sql/schema/server_stmt.rst

```diff
@@ -0,0 +1,378 @@
+
+:meta-keywords: server definition, create server, drop server, alter server, rename server
+:meta-description: Define servers in CUBRID database using create server, alter server, drop server and rename server statements.
+
+
+********************
+서버(SERVER) 정의문
+********************
+
+CREATE SERVER
+=============
+
+서버 정의
+-----------
+
+**CREATE SERVER** 문을 사용하여 새로운 서버를 생성한다. 
+생성된 서버는 **DBLINK** 기능을 이용해서  :doc:`/sql/query/select` 질의를 수행 때 원격 서버를 지정하는데 사용 할 수 있다.
+준비된 서버의 사용 방법은 `DBLINK </sql/query/select.html#dblink-clause>`_ 를 참고한다.
+
+::
+
+    CREATE SERVER <dblink_server_name> (<connect_info>) ;
+   
+        <dblink_server_name> ::= [owner_name.] server_name
+		
+        <connect_info> ::=
+                <connect_info>, <connect_item>
+                | <connect_item>						   
+        <connect_item> ::= 
+                HOST = host_string
+                | PORT = port_number 
+                | DBNAME = db_name 
+                | USER = user_name
+                | PASSWORD = [password_string]

```

Comment:
How about below?

[| PASSWORD = password_string]
[| PROPERTIES = properties_string]
[| COMMENT = server_comment_string]

Reply:
I will accept your comments and make corrections.